### ISSUE TITLE
TLS server example

### ISSUE THREAD

**Issue by jq-rs** [popravich/minihttp#33]
There is now a TLS client example, but not an example available for TLS server side. It would be awesome to have a TLS server example.

**Comment by tailhook** [comment]
Yes. It shouldn't be too hard to figure out how to do it. Contributions are welcome
